app/portainer/views/stacks/create/createstack.html

```diff
@@ -130,6 +130,12 @@
                 Indicate the path to the Compose file from the root of your repository.
               </span>
             </div>
+            <div class="form-group">
```

Comment:
This block should be located before the text description of the Compose path field.

Also, it should have a description as well:

```html
<div class="form-group">
              <span class="col-sm-12 text-muted small">
                Specify a reference of the repository. If not specified, will use the <code>master</code> branch.
              </span>
            </div>
```

Reply:
I added a bit more doc.